src/app/docs/docs.css

```diff
@@ -0,0 +1,257 @@
+@import "../../assets/css/utilities.css";
+@import "../components/avatar/avatar.css";
+@import "../components/badge/badge.css";
+@import "../../assets/css/prism-custom.css";
+
+* {
+  margin: 0px;
+  padding: 0px;
+  box-sizing: border-box;
+  font-family: "Montserrat", sans-serif;
+  font-weight: 400;
+  list-style: none;
+}
+
+:root {
+  /* --PRIMARY: #9319f7; */
+  --PRIMARY: #329986;
+  --PRIMARY-DARK: #006653;
+  /* --SECONDARY: #b563f7; */
+  --SECONDARY: #65ccb9;
+  --SECONDARY-DARK: #4c998b;
+  --WHITE: #fff;
+  --GREEN: #008000;
+  --ALIZARIN: #e92020;
+  --GRAY: #808080;
+  --CORAL: #ff7f50;
+  --BLUE-VIOLET: #8a2be2;
+  --GREEN-LIGHT: #008000;
+  --DARK-GRAY: #9e9e9e2b;
+  --DISABLED-GREY: #565454;
+
+  --STEEL-GREY: #41464b;
+
+  --ZIRCON-GREY: #e2e3e5;
+
+  --ZUMTHOR-GREY: #d3d6d8;
+
+  --FUN-GREEN-SUCCESS: #0f5132;
```

Comment:
Why have you written all these variables in block letters? Any reason?  Also, its best to name the CSS variables in a generic manner. We should not add the value (example gren, white, grey) in the variable name.

Reply:
I have renamed css variable names as per where the color was been applied like SUCCESS-MESSAGE .SUCCESS-MESSAGE color was applied on success messages. 
Is it not good convention to use block letters in css variable.

Reply:
okay.Thanks for input Puneet.